src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -6,7 +6,23 @@
     public static final String OAUTH_AUTHENTICATION_SCHEME = "OAuth";
 
     public static final int VALIDATION_RESULT_MESSAGE_MAX_LENGTH = 2000;
-
+    
+    public static class AccountingStringValidationErrorMessages {
+        public static final String ERROR_CHART_OF_ACCTS_REQUIRED = "Chart is a required field.";
```

Comment:
Normally, we would put an error message in our properties file and have a corresponding key constant used to retrieve it from there and build it with token replacement if desired. Should we follow that pattern here? Also, some of these messages look pretty similar to existing messages in the system, can we reuse them?

If not, and we decide to keep these as-is, I would probably spell out the words in the constant name to help improve readability.